src/App.js

```diff
@@ -5,6 +5,7 @@ import { StorePage } from "./Componets/StorePage/store";
 import { MyCartPage } from "./Pages/AddToCardCart/cart";
 import {WishlistPage} from './Pages/wishlispage/wishList'
 import { BrowserRouter, Routes, Route,Link} from "react-router-dom";
+import {LoginPage} from './Pages/LoginPage/Login';
```

Comment:
Keep all the pages in one index file and import it from there to keep the code concise.